### PR TITLE
add tips about addon manager

### PR DESCRIPTION
**Pull request by Leafx-code**
like title,I added some hints about `git fetch failure` and publishing addons to LLS-Addons

i am not sure that need this change, but if not, let me know and I'll turn off this pr

### CODE REVIEW

File: src/content/wiki/addons.mdx [hunk 1]
```diff
@@ -92,6 +92,12 @@ The addon manager is a [webview](https://code.visualstudio.com/api/extension-gui
 
 If you have feedback on the addon manager, please leave it in the [vscode-lua repository](https://github.com/LuaLS/vscode-lua) <Icon name="heart" group="solid" color="red"/>.
 
+To publish your own addon to addon manager,you can refer this [Github Disscussion #2408](https://github.com/LuaLS/lua-language-server/discussions/2408)
```
Comment (carsakiller) [inline src/content/wiki/addons.mdx:95]: The linked discussion doesn't really have anything to do with publishing an addon. I think this part can be removed

```suggestion
```

File: src/content/wiki/addons.mdx [hunk 2]
```diff
@@ -92,6 +92,12 @@ The addon manager is a [webview](https://code.visualstudio.com/api/extension-gui
...
+
+<Remark type="note">
+  Because the addons manager's online addons service is based on the git repository on Github, if you can't use the online addons service (Github fetch failure), check your network has access to Github.
```
Comment (carsakiller) [inline src/content/wiki/addons.mdx:98]:
```suggestion
  The addon manager uses Git to retrieve addons, make sure you have access to GitHub if you receive failures to fetch.
```

**Review by carsakiller — CHANGES_REQUESTED** [review]
I think the warning about needing a connection to GitHub could be helpful, I think it just needs to be more succinct.

File: src/content/wiki/addons.mdx (hunk 2 above)
Comment (Leafx-code) [reply src/content/wiki/addons.mdx:97]: okay,i commited it
now?

Comment (carsakiller) [reply src/content/wiki/addons.mdx:97]: I still think the remark could be reworded to be a bit shorter while still letting users know they need access to Github.

**Comment by Leafx-code** [comment]
sorry it looks like that i ignored something
now i commited this change

**Review by carsakiller — APPROVED** [review]

**Pull request merged**